knowledge_runtime/.env.example

```
@@ -0,0 +1,22 @@
+HOST=0.0.0.0
+PORT=8200
+
+# Backend URL for fetching content
+BACKEND_INTERNAL_URL=http://localhost:8000
+
+# Content fetching timeout in seconds
+CONTENT_FETCH_TIMEOUT=120
+
+# Logging configuration
+LOG_FILE_ENABLED=true
+LOG_DIR=./logs
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LOG_DIR key should go before the LOG_FILE_ENABLED key

(UnorderedKey)